Assets/Mirror/Examples/Lobbey/Scripts/OfflineGUI.cs

```diff
@@ -0,0 +1,29 @@
+﻿using UnityEngine;
+using UnityEngine.SceneManagement;
+
+namespace Mirror.Examples.NetworkLobby
+{
+    public class OfflineGUI : MonoBehaviour
+    {
+        void Start()
+        {
+            // Ensure main camera is enabled because it will be disabled by PlayerController
+            Camera.main.enabled = true;
+
+            // Since this is a UI only screen, lower the framerate and thereby lower the CPU load
+            Application.targetFrameRate = 10;
+        }
+
+        void OnGUI()
+        {
+            GUI.Box(new Rect(10, 10, 200, 130), "OFFLINE  SCENE");
```

Comment:
consider GUILayout instead of manual placing rects

Reply:
Done